### PR TITLE
[meta] Add plugin stanza to META so Fl_dynload works for Coq plugins

### PR DESCRIPTION
**Pull request by ejgallego**
This should be backported to 8.10.

### CODE REVIEW

File: META.coq.in [hunk 1]
```diff
@@ -456,6 +497,9 @@ package "plugins" (
 
     archive(byte)    = "string_notation_plugin.cmo"
     archive(native)  = "string_notation_plugin.cmx"
+
+    plugin(byte)    = "string_syntax_plugin.cmo"
+    plugin(native)  = "string_syntax_plugin.cmxs"
```
Comment (vbgl) [inline META.coq.in]: What is `string_syntax_plugin`?

**Review by vbgl — CHANGES_REQUESTED** [review]

**Comment by vbgl** [comment]
Given the “8.10.1” milestone, I can assign myself to this issue.

**Review by vbgl — APPROVED** [review]

File: META.coq.in (hunk 1 above)
Comment (ejgallego) [reply META.coq.in]: Indeed, that was a typo on my script, fixed. Thanks!

**Comment by ejgallego** [comment]
Thanks!

**Pull request merged**